library/psa_crypto.c

```diff
@@ -6481,6 +6481,28 @@ static psa_status_t psa_key_derivation_input_internal(
     return status;
 }
 
+static psa_status_t psa_key_derivation_input_integer_internal(
+    psa_key_derivation_operation_t *operation,
+    psa_key_derivation_step_t step,
+    uint64_t value)
+{
+    psa_status_t status;
+    psa_algorithm_t kdf_alg = psa_key_derivation_get_kdf_alg(operation);
+
+    {
```

Comment:
Why is this a separate block? It looks like a condition is missing, although the code is correct since no algorithm currently accepts an integer argument.

Reply:
>  It looks like a condition is missing

I am going to add the `pbkdf2_input_cost` function next therefore I had kept it as a separate block. 

Since PBKDF2 is the only algorithm which supports input cost for now should I remove this `input_integer_internal` and add it to `input_integer` function itself?

Reply:
No, that's ok. I just wanted to make sure it wasn't some accidental edit mistake in deleting a line. The code is correct, and it makes sense that it looks slightly weird since it's a foundation that will be extended soon.